### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by Eternalgratis**
Popup window created dynamically
Html created dynamically

### CODE REVIEW

File: modal.css [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+  .modal-tech {
+    justify-content: flex-start;
+    text-align: left;
+    margin: 5px;
+    gap: 10px;
+    grid-column: 1 / 3;
+  }
+
+  p {
+    /* margin: 10px; */
```
Comment (CollinsTatang) [inline modal.css]: - [ ] Please kindly remove all commented code from your codebase to make it clean and easy to maintain.

File: index.html [hunk 2]
```diff
@@ -261,7 +167,50 @@ <h2 id="greeting">Hello</h2>
...
+                    </button>
+                    <button type="submit" class="modal-btn">See Source
+                        <img src="./images/github.svg" alt="see source">
+                    </button>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <script src="./modal.js"></script>
```
Comment (CollinsTatang) [inline index.html:208]: You have done a great job so far but still, have some issues to be fixed to meet up with the project [required design.](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468)
 
- [ ] At the desktop level, when a user clicks the button to check project details the navbar covers the popup closing (X) button making it difficult for users to close the popup to disappear.
### **Take a look at your design**
yourpop-up
### **Take a look at Figma design**
figmaldetails

File: modal.js [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+// pop-up menu
+
+const proj = [
+  {
+    projectTitle: 'Multi-Post Stories',
+
+    projectSubtitle: 'Gain+Glory',
+
+    skills: ['Ruby on rails', 'CSS', 'JavaScript', 'HTML'],
+  },
```
Comment (CollinsTatang) [inline modal.js]: - [ ] Please each object should contain the following information about the project: name, description, featured image, technologies, link to live version, and link to the project source code.
```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```

File: modal.css [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+  align-items: center;
+  padding: 10px;
+  background-color: #36b37e;
+  border-radius: 5px;
+  border-style: none;
+  color: #fff;
+  margin: 5px;
+}
+
+@media only screen and (min-width: 768px) {
```
Comment (CollinsTatang) [inline modal.css:117]: - [ ] At mobile view please try to remove the hamburger menu to meet up with the required [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468)
### **Your Design**
yourM
### **Figma Design**
mobilV

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @Eternalgratis, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modal.css (hunk 4 above)
Comment (Eternalgratis) [reply modal.css:117]: Done

File: modal.js (hunk 3 above)
Comment (Eternalgratis) [reply modal.js]: I only have 3 information about my project according to my figma design

Comment (CollinsTatang) [reply modal.js]: Hi @Eternalgratis 
According to the project requirement, you have to return the project image and live_link inside the array of objects as seen above.
Thanks

**Comment by OmarRamoun** [comment]
@Eternalgratis. You forgot to push your code

**Review by OmarRamoun — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @Eternalgratis 👋

Your project is complete! ✔️ 🏁

Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟

- ✔️ Descriptive commit messages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**